.github/workflows/cron.yml

```
@@ -3,58 +3,65 @@ name: Cron
-        uses: actions/setup-python@v5
+      - name: Connect to OpenVPN
+        run: |
+          sudo apt-get install -y openvpn
+          echo "$OPENVPN_USER" | sudo tee -a /etc/openvpn/client/auth
+          echo "$OPENVPN_PASS" | sudo tee -a /etc/openvpn/client/auth
+          echo "$OPENVPN_CONFIG" | sudo tee -a /etc/openvpn/ovpn.conf
+          sudo openvpn --config /etc/openvpn/ovpn.conf --daemon
+          sleep 120
+
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v1
```

actionlint (1.7.4)

the runner of "actions/setup-python@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)